### ISSUE TITLE
404 when browsing to root context

### ISSUE THREAD

**Issue by bparry02** [redhat-cop/spring-rest#33]
The application root context tries to redirect to `/swagger-ui.html` but an error message is displayed:

```
Whitelabel Error Page

This application has no explicit mapping for /error, so you are seeing this as a fallback.
Fri Jul 19 13:17:35 UTC 2019
There was an unexpected error (type=Not Found, status=404).
Not Found
```

**Comment by bparry02** [comment]
It's possible #31 will resolve this issue but I haven't tested it yes

**Comment by bparry02** [comment]
Resolved by #38 

Root context now redirects to an API endpoint:
```
$ curl -v http://localhost:8080/
* About to connect() to localhost port 8080 (#0)
*   Trying ::1...
* Connected to localhost (::1) port 8080 (#0)
> GET / HTTP/1.1
> User-Agent: curl/7.29.0
> Host: localhost:8080
> Accept: */*
> 
< HTTP/1.1 302 Found
< Access-Control-Allow-Headers: Content-Type, Accept, X-Requested-With, remember-me
< Location: http://localhost:8080/v1/greeting
< Date: Tue, 19 Nov 2019 02:08:52 GMT
< Connection: keep-alive
< Access-Control-Allow-Credentials: true
< Content-Length: 0
< Content-Language: en-US
< Access-Control-Allow-Methods: *
< Access-Control-Max-Age: 3600
< 
* Connection #0 to host localhost left intact
```
